cli/tests/parse.rs

```diff
@@ -25,3 +25,22 @@ fn parse_with_project_lockfile() {
         .success()
         .stdout(predicate::str::contains("\"name\": \"typescript\""));
 }
+
+#[test]
+fn parse_with_project_lockfile_relative_paths() {
+    // Setup CLI with temp dir.
+    let test_cli = TestCli::builder().cwd_temp().build();
+    let temp_path = test_cli.temp_path();
+
+    // Write .phylum_project to temp dir.
+    let config = "id: 00000000-0000-0000-0000-000000000000\nname: test\ncreated_at: \
+                  2000-01-01T00:00:00.0Z\nlockfile_path: ./package-lock.json\nlockfile_type: npm";
+    fs::write(temp_path.join(".phylum_project"), config).unwrap();
+
+    // Copy lockfile to temp dir.
+    fs::copy("../tests/fixtures/package-lock.json", temp_path.join("./package-lock.json")).unwrap();
+
+    let not_contain_var_folders =
+        predicate::function(|x: &str| !x.contains("\"lockfile\": \"/private/var/folders"));
+    test_cli.cmd().args(["parse"]).assert().success().stdout(not_contain_var_folders);
```

Comment:
This test relies on `/private/var/folders`, which is specific to MacOS. To make this test work across platforms, create a temp folder with a specific name and check for that instead:

```suggestion
    // Setup CLI with temp dir.
    let tempdir = TempDir::new().unwrap();
    let sensitive_dir = tempdir.path().join("sensitive_dir_name");
    fs::create_dir_all(&sensitive_dir).unwrap();
    let test_cli = TestCli::builder().cwd(sensitive_dir.clone()).build();

    // Write .phylum_project to temp dir.
    let config = "id: 00000000-0000-0000-0000-000000000000\nname: test\ncreated_at: \
                  2000-01-01T00:00:00.0Z\nlockfile_path: ./package-lock.json\nlockfile_type: npm";
    fs::write(sensitive_dir.join(".phylum_project"), config).unwrap();

    // Copy lockfile to temp dir.
    fs::copy("../tests/fixtures/package-lock.json", sensitive_dir.join("./package-lock.json"))
        .unwrap();

    let not_contain_var_folders = predicate::str::contains("sensitive_dir_name").not();
    test_cli.cmd().args(["parse"]).assert().success().stdout(not_contain_var_folders);
```